lib/active_model/serializer/configuration.rb

```diff
@@ -26,6 +26,7 @@ def config.array_serializer
         # Make JSON API top-level jsonapi member opt-in
         # ref: http://jsonapi.org/format/#document-top-level
         config.jsonapi_include_toplevel_object = false
+        config.key_transform = nil
```

Comment:
Just we should document that. Could be surprising. Or make it a required method in every adapter by just not defining it in the Base